opentelemetry/README.md

```diff
@@ -0,0 +1,88 @@
+# OpenTelemetry-Ktor Demo
+
+## Running
+
+**Note:** You need to have [Docker](https://www.docker.com/) installed and running to run the sample.
+
+To run a sample, first, execute the following command in an `opentelemetry` directory:
```

Comment:
Edit: "To run this sample, execute the following command from the `opentelemetry` directory:"